app/models/event.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
+  validates :title, :description, :location, :date, presence: true
+
+  belongs_to :creator, class_name: 'User', foreign_key: 'creator_id'
```

Comment:
The `foreign_key` statement is not needed. Rails, due to convention, will reference User as creator_id because of the `class_name` declaration. You could remove foreign key and it should still work as intended. [OPTIONAL]